worker/instancemutater/mutater.go

```diff
@@ -5,7 +5,6 @@ package instancemutater
 
 import (
 	"fmt"
-
```

Comment:
Looks like an accidental removal. Goimports seems to be complaining about it.

Reply:
ty